WalletWasabi.Fluent/Views/CoinControl/SelectCoinsDialogView.axaml

```diff
@@ -0,0 +1,119 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:converters="clr-namespace:WalletWasabi.Fluent.Converters"
+             xmlns:generic="clr-namespace:System.Collections.Generic;assembly=System.Collections"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="650"
+             xmlns:coinControl="clr-namespace:WalletWasabi.Fluent.ViewModels.CoinControl"
+             x:DataType="coinControl:SelectCoinsDialogViewModel"
+             x:Class="WalletWasabi.Fluent.Views.CoinControl.SelectCoinsDialogView">
```

Comment:
Enable compiled bindings

Reply:
I've enabled the ones that can be enabled. 
Please, take into account that this XAML is almost the final version, and I've kept binding definitions to properties that still don't exist (will come hereafter). I prefer this than removing and readding them, in order to make things less confusing.